packages/static-wado-plugins/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@ohif/static-wado-plugins",
+  "version": "0.5.1",
+  "description": "Plugins to support the static-wado project",
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "StaticWado",
+    "DICOM",
+    "DICOMweb"
+  ],
+  "author": [
+    "Bill Wallace <bill.wallace@radicalimaging.com>",
+    "Rodolfo Costa <rodolfo.costa@radicalimaging.com>"
+  ],
+  "engines": {
+    "node": ">=14.18.1",
+    "npm": ">=6.14.15",
+    "yarn": ">=1.22.4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OHIF/static-wado",
+    "directory": "packages/static-wado-plugins"
+  },
+  "bugs": {
+    "url": "https://github.com/OHIF/static-wado/issues"
+  },
+  "homepage": "https://github.com/OHIF/static-wado#readme",
+  "license": "ISC",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "test": "jest --config ./jest.config.js",
+    "build": "echo \"No build yet\" && exit 0",
+    "link:exec": "npm link",
+    "lint": "npx eslint",
+    "lint:fix": "npx eslint --fix"
+  },
+  "jest": {
+    "verbose": true
+  },
+  "dependencies": {
+    "@ohif/static-wado-util": "^0.5.1",
+    "config-point": "0.3.9"
```

Comment:
looks like there is already two different versions for config-point, can you use this PR and apply the same version for all 3 (0.3.9 or ^0.3.9)?